### PR TITLE
Add Coding Standards page

### PR DESCRIPTION
**Pull request by bcremer**

### CODE REVIEW

File: source/developers-guide/coding-standards/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Coding Standards
+github_link: developers-guide/coding-standards/index.md
+indexed: true
+---
+
+The shopware team uses the following coding standards. All contributions to Shopware as well a plugins and customizations should also use these standards.
```
Comment (mitelg) [inline source/developers-guide/coding-standards/index.md]: as well as

File: source/developers-guide/coding-standards/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+indexed: true
+---
+
+The shopware team uses the following coding standards. All contributions to Shopware as well as plugins and customizations should also use these standards. 
+
+## PHP Coding Standard
+For PHP Code all contributions should use the [PSR-1: Basic Coding Standard](http://www.php-fig.org/psr/psr-1/) and [PSR-2: Coding Style Guide](http://www.php-fig.org/psr/psr-2/).
+
+## JavaScript Coding Standard
+Please see our dedicated page for the [JavaScript Coding Standard](/designers-guide/javascript-coding-style/).
```
Comment (klarstil) [inline source/developers-guide/coding-standards/index.md]: Use `{{ site.url }}` to get the full url of the page

Comment (bcremer) [inline source/developers-guide/coding-standards/index.md]: Why? The relative URL seems fine for me. We use relative URLs in various places already.

Comment (klarstil) [inline source/developers-guide/coding-standards/index.md]: I like to have the application future proof. You never know if you have to move the application in a sub directory sometime in the future, that's why.

Comment (bcremer) [inline source/developers-guide/coding-standards/index.md]: Fair enough. But the `{{ site.url }}` Twig tag is not compatible with the markdown link syntax so I will leave it for now.
If we ever have to move the devdocs to a subdirectory we have to change a whole lot of links anyway (regex ahead).

**Pull request merged**